lib/widgets/button.dart

```diff
@@ -1,12 +1,22 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart';
 
+/// The role of the button.
 enum ViamButtonRole {
+  /// Standard button. Light gray background, dark gray foreground (inverse when in dark mode)
   primary,
+
+  /// Inverse of the primary button, Dark gray background, light gray foreground (inverse when in dark mode)
   inverse,
+
+  /// A button to indicate a successful operation, will result in a green color scheme
   success,
-  danger,
-  warning;
+
+  /// A button to indicate a warning state, will result in an amber color scheme
+  warning,
```

Comment:
What is an example of something that is in a warning state? Just trying to understand the use case

Reply:
Maybe an action that's not fully destructive, but just a warning. Maybe something like clearing logs in the future?